include/fp-folding.h

```diff
@@ -57,6 +57,39 @@ typedef float float32_t;
 typedef double float64_t;
 typedef long double float128_t; /* 128 bits in memory, format host-dependent */
 
+#if !defined(_WIN64)
```

Comment:
```suggestion
#if !defined(_MSC_VER)
```
or `_WIN32`. don't know about mingw. I would assume since it is gcc it has a complex type.

Reply:
@Neumann-A Does Adam's latest change look good to you?

Reply:
@Neumann-A If this looks okay to you, we will merge this PR.

Reply:
Looks ok to me but as i said i dont know about mingw.

Reply:
Thanks. We'll merge this.